src/util.ts

```diff
@@ -999,6 +965,72 @@ export class Util {
         }
         return result;
     }
+
+    /**
+     * Convert a token into a BscType
+     */
+    public tokenToBscType(token: Token) {
```

Comment:
This is a helpful utility function, which converts various token types into a BscType.